### DISCUSSION TITLE
Modeling IAB workshop artifacts

### DISCUSSION THREAD

**Discussion by rjsparks** [ietf-tools/datatracker#6750] [General]
From a discussion with @mirjak:

It is desirable to allow submitters to upload workshop papers directly to the datatracker, and provide the workshop program committee a workflow that results in accepted papers being published. Papers that are not yet (or ever) published would not be shown to anyone other than the publication committee.

One thought is to model these as one or more new Document types, and to have a submission/review process built around them.

One edge to discuss:

* How sensitive are the non-accepted papers? Would it be problematic if they were available to developers as part of a developer database dump? (Generally, adding things that need to be kept confidential to the datatracker turn out to be the most expensive/problematic additions over time).

There are conference paper review tools that are relatively inexpensive - would using hosted instances of hotcrp be a better/worse solution (with only the published papers being captured in the datatracker)?

We identified that at the end, the final results would need to live at long-term-stable URLs. Designing that is a separate conversation.

cc: @cindymorgan

**Comment by mirjak** [comment]
As we follow a very light way review process, we don't really need a full reviewing tool like hotcrp. we discussed that in the past (and we could discuss it again) but back then the feeling was that is was rather overhead for us.

I don't think we really provide any promises about confidentiality. We usually don't get papers like for conferences where if a paper is rejected people will resubmit it somewhere else. I think we can address that by just adding a sentence to the call saying that we will not treat papers as confidential.
